rs/sns/governance/api/src/lib.rs

```diff
@@ -8,3 +10,8 @@ pub fn format_full_hash(hash: &[u8]) -> String {
         .collect::<Vec<_>>()
         .join("")
 }
+
+/// Formats `journal.entries` as JSON.
+pub fn serialize_journal_entries(journal: &UpgradeJournal) -> Result<String, String> {
```

Comment:
This function is moved here because it's very helpful for Rust-based SNS Gov API clients, e.g., wiring debuggable integration tests.

Reply:
Then the clients would need to add another direct dependency, avoiding which is one of the points of having the function here.